scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -71,7 +71,7 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
     // Drop the period for the extenion.
     const ext = parsedPath.ext.slice(1);
     // For jsx or js files, we want to drop the extention for including them, because we don't know if the file will be transpiled.
-    const fileName = (ext === 'jsx' || ext === 'js') ? parsedPath.name : parsedPath.base;
+    const fileName = ('tsx' || ext === 'ts' || ext === 'jsx' || ext === 'js') ? parsedPath.name : parsedPath.base;
```

Comment:
Not sure where this was missing "ext" === "tsx" but Im sure this is the issue

Reply:
build failed to deploy with:
<img width="986" alt="Screen Shot 2019-09-04 at 3 59 08 PM" src="https://user-images.githubusercontent.com/14099737/64291112-fd7f4d80-cf2c-11e9-849a-aabe8694dde8.png">